### ISSUE TITLE
Favicon is not working in our repository

### ISSUE THREAD

**Issue by bartekpacia** [invertase/docs.page#241]
Repository concerned: https://github.com/leancodepl/patrol

I think I have set the correct `favicon` in `docs.json` and reference the correct asset, but the favicon is still the default one. I'd be very grateful if somebody of you, dear creators, took a look at our repository and told me what I'm doing wrong 😅

**Comment by bartekpacia** [comment]
Huh, social preview is not working as well :(

**Comment by cabljac** [comment]
Hey, thanks for raising this!

I took a look at your config file, and `favicon` seems to be set correctly, which means this is probably a bug. For the social preview image, try changing it to a URL of the image and see if that fixes things, i.e 
`
https://github.com/leancodepl/patrol/blob/master/docs/assets/preview.jpg?raw=true
`
As a workaround until a fix comes in, changing `favicon` to `logo` might work for you.

I'll look into this ASAP though

**Comment by cabljac** [comment]
Also note that twitter and other sites will cache social previews, so any changes may take a while to appear.

**Comment by bartekpacia** [comment]
Thanks a lot for your help, will try out the workarounds tomorrow.

**Comment by Ehesp** [comment]
@bartekpacia Might need to re-added, but `favicon` should work now in your config.

**Comment by bartekpacia** [comment]
Yeah, it works. Done in https://github.com/leancodepl/patrol/pull/744.

It'd be also great if we were able to specify `faviconDark` :)

**Comment by Ehesp** [comment]
Ah that needs JavaScript to do that from what I remember - I'll take a look.

**Comment by Ehesp** [comment]
Oh, or something like 

```html
<link rel="icon" href="/favicon.ico?dark"  media="(prefers-color-scheme:dark)">
<link rel="icon" href="/favicon.ico?light" media="(prefers-color-scheme:light)">
```
